EIPS/static_call.md

```diff
@@ -0,0 +1,40 @@
+## Preamble
+
+    EIP: to be assigned
+    Title: New opcode STATIC_CALL
+    Author: Vitalik Buterin <vitalik@ethereum.org>, Christian Reitwiessner <chris@ethereum.org>;
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-02-13
+
+## Simple Summary
+
+To increase smart contract security, this proposal adds a new opcode that can be used to call another contract (or itself) while disallowing any modifications to the state during the call (and its subcalls, if present).
+
+## Abstract
+
+
+## Motivation
+
+
+## Specification
+
+Opcode: `0xfa`.
+
+`STATIC_CALL` functions equivalently to a `CALL`, except it takes 6 arguments not including value, and calls the child with a `STATIC` flag on. Any calls, static or otherwise, made by an execution instance with a `STATIC` flag on will also have a `STATIC` flag on. Any attempts to make state-changing operations inside an execution instance with a `STATIC` flag on will instead throw an exception. These operations include nonzero-value calls, creates, `LOG` calls, `SSTORE`, `SSTOREBYTES` and `SUICIDE`.
```

Comment:
So far we haven't separated words in the mnemonics: `CALLCODE` and `DELEGATECALL`.  Maybe `STATICCALL` is more consistent.